Density/app/src/main/java/org/cornelldti/density/density/facilitydetail/FacilityInfoPage.kt

```diff
@@ -493,6 +491,16 @@ class FacilityInfoPage : BaseActivity() {
                 adapter = menuItemListViewAdapter
             }
 
+            // This is where the operating hours for the selected meal of day is set!
+            if(availableMenus.isNotEmpty() && opHoursStrings.isNotEmpty()) {
+                menuHours.text = opHoursStrings[availableMenus.indexOf(mealOfDay)]
+                clock_image.visibility = View.VISIBLE
+            }
+            else {
+                menuHours.text = ""
+                clock_image.visibility = View.INVISIBLE
+            }
```

Comment:
I haven't been able to test the edge cases, but you almost always want to get visibility to `View.GONE` so it doesn't take up layout space